samples/samples_common.py

```diff
@@ -0,0 +1,29 @@
+# Common functions for sample code.
+
+import sys
+
+from getpass import getpass
+
+try:
+    get_input = raw_input
+except NameError:
+    get_input = input
+
+
+def get_user_details():
```

Comment:
That's fair. I don't know why it didn't occur to me that I could pass in sys.argv as a list to make it clearer.